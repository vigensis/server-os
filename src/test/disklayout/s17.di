SCSI	disk0	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk1	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk2	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk3	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk4	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk5	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk6	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk7	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk8	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk9	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	diska	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	diskb	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	diskc	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	diskd	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	diske	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	diskf	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk10	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
