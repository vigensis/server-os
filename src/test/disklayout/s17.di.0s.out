fatal error: invalid number of spares (0) for this disk configuration
